lib/setup.py

```diff
@@ -52,21 +52,12 @@
     # upper bound on it.
     "pyarrow>=6.0",
     "pympler>=0.9, <2",
-<<<<<<< HEAD
```

Comment:
I merged the latest streamlit release into our fork, but forgot to fix this merge conflict before committing that.